.pre-commit-config.yaml

```diff
@@ -9,13 +9,15 @@
 #
 #
 default_language_version:
-  python: python3.8
+  python: python3.10
 
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.0.276
+    rev: v0.1.8
```

Comment:
https://pypi.org/project/ruff/ suggests that this is at `0.2.2` now.

Reply:
Yes, but they have a new release every few weeks.
I prefer to use the same version across all our projects, and update only once a year or so.